mule-packager/src/test/java/org/mule/tools/api/verifier/policy/PolicyYamlValidationTest.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.verifier.policy;
+
+import static java.lang.String.join;
+import static java.nio.file.Paths.get;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class PolicyYamlValidationTest {
+
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
+  @Test
+  public void normalYamlParsesCorrectly() throws IOException {
+    testYaml("caching-yaml-example.yaml");
+  }
+
+  @Test
+  public void minimalYamlParsesCorrectly() throws IOException {
```

Comment:
And between this and the next?

Reply:
this one is a yaml that has only the minimum amount of fields that should be in the file. And the next one is the same as this but with some extra random fields, and the test verifies that they are ignored.